app/ceom/aboutus/views.py

```diff
@@ -18,9 +18,9 @@ def people(request):
     
     return render(request, 'aboutus/people.html', context=data)
 
-def index(request):
+def publications(request):
+    print("Hi:)")
     pubs = Publication.objects.all().order_by('-date')
-    #pubs.reverse()
 
     if 'type' in request.GET:
```

Comment:
I'm curious about what happens when the "type" part of the request is missing. Does the page still work?

Reply:
Technically yes it works and displays the data, but it doesn't separate the journals, books, and presentations. Without it, the data is the same for every page.

Reply:
Actually, sorry I take that back it does work